src/charm.py

```diff
@@ -183,68 +218,10 @@ def _restart_service(self):
         self.container.start(PEER)
```

Comment:
container name is not guaranteed to be peer relation name

Reply:
I have the service `PEER` defined in the layer... 

I followed [this](https://juju.is/docs/sdk/workloads#heading--kubernetes-charms):

```python
        # If the 'pause' service is currently running in the container, stop it
        if service.is_running()
            container.stop("pause")
        # Start/restart the 'pause' service in the container
        container.start("pause")
        logging.info("Restarted pause service")
```